Assets/MixedRealityToolkit/InputSystem/Utilities/Enums/OculusControllerInputType.cs

```diff
@@ -1,46 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-
-using System;
-
-namespace Microsoft.MixedReality.Toolkit.InputSystem.Utilities
-{
-    [Flags]
-    public enum OculusControllerInputType
-    {
-        None                           = 0,
-        ButtonOnePress                 = 1 << 0,
```

Comment:
Maybe we should take some of these Enum Values for the Device Input Type (yay thought of a better name)